<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DModBusSettings</class>
 <widget class="QDialog" name="DModBusSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressSearch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchDevices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBaudeRate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>115200</string>
   </property>
   <property name="maxVisibleItems">
    <number>10</number>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelSearch">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>90</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>Время ожидания
поиска устройств
(секунды)</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>130</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Скорость передачи</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadDevices">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Загрузка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;Поиск&quot; выполняет поиск новых устройств
&quot;Загрузка&quot; использует сохранённый список устройств</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
